services/storage/src/simcore_service_storage/api/v0/openapi.yaml

```diff
@@ -265,6 +289,34 @@ paths:
                 $ref: '#/components/schemas/FileMetaEnvelope'
         default:
           $ref: '#/components/responses/DefaultErrorResponse'
+    delete:
+      summary: Removes a file's meta data entry
+      operationId: delete_file_meta_data
+      parameters:
+        - name: fileId
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: location_id
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: user_id
+          in: query
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
```

Comment:
this should be a 204. and actually I am wondering if the "delete" file entry in the storage should not do a proper cleanup if the entry is only in the database?

Reply:
Agreed. all our deletes are 204